tests/components/application-connector/Makefile

```diff
@@ -2,24 +2,43 @@ NAMESPACE ?= test
 GOPATH ?= $(shell go env GOPATH)
 
 GATEWAY_TEST_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/gateway-test:$(DOCKER_TAG)"
+VALIDATOR_TEST_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/app-connectivity-validator-test:$(DOCKER_TAG)"
 MOCK_APP_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/mock-app:$(DOCKER_TAG)"
 
-.PHONY: release test-os test-skr test image publish clean image-gateway-test publish-gateway-test test-gateway test-gateway-debug image-mock-app publish-mock-app
+.PHONY: release test image publish clean image-gateway-test publish-gateway-test test-gateway test-gateway-debug image-mock-app publish-mock-app
 
-test: test-gateway
-image: image-gateway-test
-clean: clean-gateway-test
+test: test-gateway test-validator
```

Comment:
Shouldn't it be a different command? I mean, we want this test to be run separately from the Gateway tests. Correct me if I'm wrong here